TSE/CCExtensions.cpp

```diff
@@ -1656,18 +1656,34 @@ static PRIMITIVEPROCDEF g_Extensions[] =
 			"property (all)\n\n"
 
 			"   'ascended\n"
-			"   'category\n"
+			"   'category -> 'beam | 'effect | 'marker | 'missile | 'ship | 'station\n"
+			"   'commsKey\n"
+			"   'currency -> currency type UNID\n"
```

Comment:
Using `->` here is not consistent with the other help text which exclusively uses `->` in the function signature. It might be clearer to follow the style of e.g. msnGetProperty or sysFindObject and use space alignment:
```
"   'currency          Currency type UNID\n"
```